examples/widgets/slider_demo.py

```diff
@@ -5,63 +5,83 @@
 
 Using the slider widget to control visual properties of your plot.
```

Comment:
I find this statement to be slightly confusing/misleading. Thoughts on the following?:
1. removing it
2. replacing with something like:
```suggestion
`~.Slider` widgets can be used to dynamically update any aspect of a plot by use of a callbacks.
```
(can `plot` be taken in the colloquial sense here, or is that meaning reserved for `plt.plot`?)

Reply:
This can be removed. It does not contain significant additional information that is not already clear from the following text.

*Side remark:* We also use "plot" colloquially in the above sense throughout the docs.